src/test/java/website/frontrow/util/PointTest.java

```diff
@@ -143,6 +143,18 @@ public void divideTest()
     }
 
     /**
+     * Test dividing numbers by a divisor.
+     */
+    @Test
+    public void subtractTest()
+    {
+        Point a = new Point(2, 0);
```

Comment:
Not an abbreviation, just had no clue what to name the other point.
Because pretty much any name works.

The current name signifies is that in a - b, this is the value a.
Any suggestions?